model/QtiJsonItemCompiler.php

```diff
@@ -124,13 +130,30 @@ protected function deployQtiItem(
             return new common_report_Report(
                 common_report_Report::TYPE_ERROR, $e->getUserMessage()
             );
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
             return new common_report_Report(
                 common_report_Report::TYPE_ERROR, $e->getMessage()
             );
         }
     }
 
+    /**
+     * Convert internal parameters to json if needed
+     * @param $data
+     * @return array
+     * @throws common_exception_Error
+     */
+    protected function convertXmlAttributes($data) {
+
+        if (is_array($data) && array_key_exists('core', $data)
+            && array_key_exists('apipAccessibility', $data['core'])) {
+
```

Comment:
according psr 5.1 : 
if (
    $expr1
    && $expr2
) {

Reply:
if core will not array we will get error: so we can add is_array($data['core']) checking or replace 
 array_key_exists('core', $data) && array_key_exists('apipAccessibility', $data['core']) to isset($data['core']['apipAccessibility'])

Reply:
I added condition that data['core'] is array , I don't want to use isset as it is not checking that key exists, it just checks that result is not null